src/tests/ftest/recovery/ddb.py

```diff
@@ -170,11 +84,13 @@ def test_ls(self):
         errors = []
 
         # Insert objects with API.
-        object_count = self.params.get("object_count", "/run/*")
-        dkey_count = self.params.get("dkey_count", "/run/*")
-        akey_count = self.params.get("akey_count", "/run/*")
-        self.insert_objects(
-            object_count=object_count, dkey_count=dkey_count, akey_count=akey_count)
+        object_count = 5
+        dkey_count = 2
+        akey_count = 1
+        insert_objects(
+            context=self.context, container=self.container, object_count=5,
+            dkey_count=2, akey_count=1, base_dkey=self.random_dkey,
```

Comment:
Why not pass the variables?
```suggestion
        object_count = 5
        dkey_count = 2
        akey_count = 1
        insert_objects(
            context=self.context, container=self.container, object_count=object_count ,
            dkey_count=dkey_count, akey_count=akey_count , base_dkey=self.random_dkey,
```

Reply:
Fixed